library/src/blas1/rocblas_iamax.cpp

```diff
@@ -0,0 +1,71 @@
+/* ************************************************************************
+ * Copyright 2018-2019 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+
+#include "rocblas_iamax.hpp"
+#include "rocblas_reduction_impl.hpp"
+
+namespace
+{
```

Comment:
Unnamed namespace is not required, since all of its contents are `static`.

Either use `static`, or an unnamed namespace. Using both is redundant and longer than necessary.

Reply:
okay